sunpy/lightcurve/__init__.py

```diff
@@ -9,6 +9,14 @@
 __author__ = "Keith Hughitt"
 __email__ = "keith.hughitt@nasa.gov"
 
+import warnings
+from sunpy.util.exceptions import SunpyDeprecationWarning
```

Comment:
because of this import this file needs an `__all__` list, it's breaking the docs build.

(Maybe an easier out would be importing it with a leading `_`

Reply:
See I thought that but WCS doesn't have an `__all__` in the `__init__` file.
I will just do that.